src/test/java/com/kalessil/phpStorm/phpInspectionsEA/api/JsonEncodingApiUsageInspectorTest.java

```diff
@@ -38,4 +38,21 @@ public void testIfFindsErrorsHandlingPatterns() {
             myFixture.checkResultByFile("testData/fixtures/api/json-encoding-errors-handling.fixed.php");
         }
     }
+    public void testItRecognizesNameIdentifierFlags() {
+        final PhpLanguageLevel level = PhpLanguageLevel.parse("8.0");
+        if (level != null && level.getVersionString().equals("8.0")) {
```

Comment:
Shoudln't we test for everything greater than or equal to 8.0? (8.1, 8.2, etc)

Reply:
Well, if it works in 8.0 it should also work in higher versions. I believe this will just generate an php version 8.0 PhpProjectConfigurationFacade